react-exercises/modal-react/src/Components/Modal/styles.scss

```
@@ -19,8 +19,37 @@
   }
   .modal {
     margin: 0 auto;
-    width: auto;
     display: inline-block;
+    border: 1px solid gray;
+    opacity: 0.9;
+    .bg-white{
```

please format properly
```suggestion
    .bg-white {
```